default/AI/FreeOrionAI.py

```diff
@@ -68,8 +69,8 @@ def startNewGame(aggression=fo.aggression.aggressive):  # pylint: disable=invali
     universe = fo.getUniverse()
     if planet_id is not None and planet_id != -1:
         planet = universe.getPlanet(planet_id)
-        new_name = random.choice(_capitols.get(aggression, "").split('\n')).strip() + " " + planet.name
-        print "Capitol City Names are: ", _capitols
+        new_name = " ".join([random.choice(_capitals.get(aggression, []) or [" "]).strip(), planet.name])
```

Comment:
I'll also point out that the entire purpose of having the AIs rename their capitals is to hint/signal their aggression level to the human player(s), and so even if I had been making more substantive changes to the naming it would still fit quite well in a PR about basic diplomacy.